### DISCUSSION TITLE
Is it possible to create Resource monitor with warehouses ?

### DISCUSSION THREAD

**Discussion by jaikumarkunduru01** [Snowflake-Labs/terraform-provider-snowflake#809] [Q&A]
Hello Team.

Sorry for copy-pasting the same question, I am a bit stuck at the moment. Any help would be great and appreciated.

 I would like to terraform resource monitors of warehouses. As it seems in the docs and in the code (I don't know GO so maybe I am missing something), the resource "snowflake_resource_monitor" has no warehouse(s) arguments. Thus, I am assuming that we can just manage resource monitors that monitor the entire account. However, the description of the argument end_timestamp is:
The date and time when the resource monitor suspends the assigned warehouses.

Which explicitly says warehouses, so I am a little confused here.

In conclusion, is it possible to monitor warehouses with snowflake_resource_monitor?
Thank you very much in advance
